components/teams-service/storage/postgres/postgres.go

```diff
@@ -335,20 +334,9 @@ func (p *postgres) PurgeUserMembership(ctx context.Context, userID string) ([]uu
 	for rows.Next() {
 		var teamID uuid.UUID
 		if err := rows.Scan(&teamID); err != nil {
-			return nil, err // TODO
-		}
-		teamsIDsUpdated = append(teamsIDsUpdated, teamID)
-	}
-	// touch affected teams
-	for _, teamID := range teamsIDsUpdated {
-		_, err := p.touchTeam(ctx, tx, teamID)
-		if err != nil {
 			return nil, p.processError(err)
 		}
-	}
-	err = tx.Commit()
-	if err != nil {
-		return nil, p.processError(err)
+		teamsIDsUpdated = append(teamsIDsUpdated, teamID)
 	}
 
 	return teamsIDsUpdated, nil
```

Comment:
Maybe I'm misunderstanding, but I think this function and a number of functions in this file should be calling rows.Err() to do a final err check before returning (see https://golang.org/pkg/database/sql/#Rows.Next). Might make sense to just let it go in this PR and do a sweep in a follow up PR.

Reply:
> Next prepares the next result row for reading with the Scan method. It returns true on success, or **false if there is no next result row or an error happened while preparing it**. Err should be consulted to distinguish between the two cases.

Yup, good point. I'd also lean towards a follow-up sweep 😃

Reply:
⏩ #696